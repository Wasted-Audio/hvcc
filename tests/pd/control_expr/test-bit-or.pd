#N canvas 1137 654 450 300 12;
#X obj 52 25 loadbang;
#X obj 52 160 print;
#X msg 52 85 2.3;
#X obj 164 162 print;
#X msg 106 85 -7.3;
#X obj 52 49 t b b;
#X obj 52 125 expr $i1 | 3;
#X obj 164 125 expr 11 | $i1;
#X connect 0 0 5 0;
#X connect 2 0 6 0;
#X connect 4 0 7 0;
#X connect 5 0 2 0;
#X connect 5 1 4 0;
#X connect 6 0 1 0;
#X connect 7 0 3 0;
