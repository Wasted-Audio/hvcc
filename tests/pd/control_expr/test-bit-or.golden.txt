[@ 0.000] print: -5
[@ 0.000] print: 3
